site/get/index.md

```diff
@@ -40,6 +40,8 @@ Or you can build an LTS image from source.
 
 ## Legacy LTS releases
 
+As a courtesy, we offer images of legacy LTS releases. To access all available legacy LTS releases, kindly complete the form below. Upon submission, you will receive an email containing links to download the images.
```

Comment:
I don't see a reason why this cannot be in a single commit with the form. It's a part of the same atomic change.